corehq/apps/hqwebapp/static/hqwebapp/js/select2_knockout_bindings.ko.js

```diff
@@ -10,6 +10,28 @@ hqDefine("hqwebapp/js/select2_knockout_bindings.ko", [
     ko,
     DOMPurify
 ) {
+    /**
+     * Converts the bound element to a select2 widget, using the element's existing
+     * options as data. This does not allow the options to change dynamically.
+     * The binding's value is ignored. This is useful for select2s that are part of
+     * a knockout-controlled UI but don't need dynamic options.
+     */
+    ko.bindingHandlers.staticSelect2 = function () {
+        var self = {};
+
+        self.init = function (element) {
+            $(element).select2({
+                width: "element",
+            });
+        };
+
+        self.update = function (element, valueAccessor, allBindings) {
+            // Do nothing, this should never get called
```

Comment:
should this throw or something, or is it fine?

Reply:
I'd rather err on the side of the dropdown not getting updated than risk breaking the page.

Reply:
This probably shouldn't be a binding at all, I don't know what the knockout-y way is to handle "execute some code when this element is rendered."